cax/main.py

```diff
@@ -337,5 +338,377 @@ def status():
+                        help="Run all tasks just one, then exits")
+    parser.add_argument('--config', action='store', type=str,
+                        dest='config_file',
+                        help="Load a custom .json config file into cax")
+    parser.add_argument('--run', type=int,
+                        help="Select a single run")
+    parser.add_argument('--from-run', dest='from_run', type=int, 
+                        help="Choose: run number start")
+    parser.add_argument('--to-run', dest='to_run', type=int, 
+                        help="Choose: run number end")
+    parser.add_argument('--log-file', dest='logfile', type=str, default='massive_rucio.log',
+                        help="Specify a certain logfile")
+    
+
+    args = parser.parse_args()
+
+    run_once = args.once
+
+    #Check on from-run/to-run option:
+    beg_run = -1
+    end_run = -1
+    run_window = False
+    if args.from_run == None and args.to_run == None:
+      pass
+    elif (args.from_run != None and args.to_run == None) or (args.from_run == None and args.to_run != None):
+      logging.info("Select (tpc) runs between %s and %s", args.from_run, args.to_run)
+      logging.info("Make a full selection!")
+    elif args.from_run != None and args.to_run != None and args.from_run >= args.to_run:
+      logging.info("The last run is smaller then the first run!")
+    elif args.from_run != None and args.to_run != None and args.from_run < args.to_run:
+      logging.info("Start (tpc) runs between %s and %s", args.from_run, args.to_run)
+      run_window = True
+      beg_run = args.from_run
+      end_run = args.to_run
+    
+    
+    #configure cax.json
+    config_arg = ''
+    if args.config_file:
+        if not os.path.isfile(args.config_file):
+            logging.error("Config file %s not found", args.config_file)
+        else:
+            logging.info("Using custom config file: %s",
+                         args.config_file)
+            config.set_json(args.config_file)
+            config_arg = '--config ' + os.path.abspath(args.config_file)
+      
+    # Setup logging
+    log_path = {"xe1t-datamanager": "/home/xe1ttransfer/rucio_log",
+               "midway-login1": "/home/bauermeister/rucio_log"}
+    
+    if not os.path.exists(log_path[config.get_hostname()]):
+        os.makedirs(log_path[config.get_hostname()])
+    cax_version = 'massive_ruciax_v%s - ' % __version__
+    logging.basicConfig(filename="{logp}/{logf}".format(logp=log_path[config.get_hostname()], logf=args.logfile),
+                        level=logging.INFO,
+                        format=cax_version + '%(asctime)s [%(levelname)s] ' '%(message)s')
+    
+    # Check Mongo connection
+    config.mongo_password()
+    
+
+
+    # Establish mongo connection
+    collection = config.mongo_collection()
+    sort_key = (('number', -1),
+                ('detector', -1),
+                ('_id', -1))
+
+    #Construct the basic bash script(s):
```

Comment:
Job scripts are created in config.py now, which also takes care of site dependence. (I guess should be fixed after fixing merge conflicts.)